src/summary/patient-chart-summary.css

```diff
@@ -5,3 +5,11 @@
 :global(.omrs-breakpoint-gt-phone) .patientSummary {
   padding: 1.25rem 1.5rem 0 1.5rem;
 }
+
+body {
```

Comment:
This css rule should either go into the styleguide or should be removed. The reason is that this impacts the singular `<body>` element that the entire DOM and all the single-spa applications use. Putting it into esm-patient-chart means that the body would not have 0 margin before the patient chart is loaded, but then it *would* have 0 margin after you navigate to the patient chart. With css-loader (and css modules), once a css file is loaded, it is never unmounted. So then this margin 0 would apply to the body even after patient-chart's single-spa application is unmounted.

Reply:
wow, that's interesting. I'll make the required changes